mitodl/mit-open
# Fix dialog spacing + reset AddToListDialog

### What are the relevant tickets?
This is a followup to https://github.com/mitodl/mit-open/pull/1463

### Description (What does it do?)
This does two things:
1. Fixes some spacing issues... Notably the bottom of the dialog
2. Fixes a bug where:
  1. Open the add to list dialog. ..... call this X
  2. select some checkboxes .... call this Y
  3. do not save. Close the dialog
  4. re-open ... you get Y not X

### Screenshots (if appropriate):
 
**Before** / **After**

<img width="300" alt="Screenshot 2024-08-29 at 1 37 38 PM" src="https://github.com/user-attachments/assets/8a7dbaae-2940-497b-ae42-5a61787e4369">  <img width="300" alt="Screenshot 2024-08-29 at 1 37 15 PM" src="https://github.com/user-attachments/assets/1e55b014-dded-404d-878b-5726dedcd292">


### How can this be tested?
1. View other dialogs. They should in general be unchanged (with the exception of the 4px gap mentioned below).
2. View the add to list dialog. Try to repeat bug described above. You should get the initial state ("X")
